Content.Shared/SurveillanceCamera/SharedSurveillanceCameraSystem.cs

```diff
@@ -0,0 +1,16 @@
+using Robust.Shared.GameStates;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.SurveillanceCamera;
+
+[Serializable, NetSerializable]
+public enum SurveillanceCameraVisuals
+{
+    Key,
+    Active,
+    InUse,
+    Disabled,
+    // Reserved for future use
+    Xray,
+    Emp
+}
```

Comment:
this getting used for the layer key, appearance data key, and appearance data value is kinda ass